tests/queries/0_stateless/02887_byteswap.sql

```diff
@@ -0,0 +1,34 @@
+SELECT byteSwap(0);
```

Comment:
Negative tests would be nice, e.g. what happens when there is > 1 argument, what happens when the inputs are of type != 8-64-integers (e.g. `Strings`s). Expected errors can be checked with the `serverError` syntax (grep for it in tests/queries/0_stateless/).

Reply:
Added.